src/components/listitems/ListItemHeader.tsx

```diff
@@ -125,7 +125,10 @@ export const ListItemHeader = ({
       accessible={endElement === undefined ? true : false}
       accessibilityLabel={listItemAccessibilityLabel}
     >
-      <View style={IOListItemStyles.listItemInner}>
+      <View
+        style={IOListItemStyles.listItemInner}
+        importantForAccessibility={endElement ? "auto" : "no-hide-descendants"}
```

Comment:
maybe we should handle `elementsHiddenForAccessibility` too in order to have the same behaviour both in iOS and Android, what do you think?

Reply:
Perfect! 👌🏻